subworkflows/local/prepare_genome.nf

```diff
@@ -34,6 +35,16 @@ workflow PREPARE_GENOME {
             ch_versions = ch_versions.mix(SAMTOOLS_FAIDX.out.versions)
         }
 
+        if ( params.variant_catalog && file(params.variant_catalog, checkIfExists:true) ) {
+            ch_variant_catalog = file(params.variant_catalog)
+        } else {
+            if ( params.genome == 'GRCh38' ) {
+                ch_variant_catalog = file("https://raw.githubusercontent.com/nf-core/test-datasets/raredisease/testdata/reference/variant_catalog_grch38.json")
+            } else {
+                ch_variant_catalog = file("https://raw.githubusercontent.com/nf-core/test-datasets/raredisease/testdata/reference/variant_catalog_grch37.json")
```

Comment:
Looks good! Hmm, I'm thinking if we should abstract this away from the workflow + provide a solution similar to `test-datasets/modules` where: 1) add param in nextflow.config specifying the base dir to `test-datasets/raredisease` 2) add a config referencing base dir + this file _similar to genomes.config_ 3) reference the key here 🤔